examples/tutorials/setup_poc.ipynb

```diff
@@ -34,10 +36,47 @@
    "id": "87989e99",
    "metadata": {},
    "source": [
-    "### Preparing the POC environment\n",
-    "Before running POC mode, there are a couple important environment variables that should be set.\n",
+    "> If you are the NVFLARE user prior to 2.4.0, you might noticed two things\n",
```

Comment:
In my opinion, this is not really a good place to add this extra information. This makes sense in release notes for 2.4, but it becomes excessive in this notebook and will be outdated and not as useful the further we are beyond 2.4.0.

Reply:
These two things we could move it to release notes?